pkg/controller/plan/adapter/ovirt/builder.go

```diff
@@ -485,6 +486,13 @@ func (r *Builder) mapDisks(vm *model.Workload, persistentVolumeClaims []core.Per
 	object.Template.Spec.Domain.Devices.Disks = kDisks
 }
 
+func (r *Builder) mapDevices(vm *model.Workload, object *cnv.VirtualMachineSpec) {
```

Comment:
this name is too generic since other devices are handled by other function, how about:
```suggestion
func (r *Builder) mapTpm(vm *model.VM, object *cnv.VirtualMachineSpec) {
```

Reply:
I preferred to leave it as workload, to align with all the methods.

Reply:
sure, I don't remember how the type changed - I meant to rename the function